src/Psl/IO/Internal/open.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\IO\Internal;
+
+use Psl;
+use Psl\Internal;
+
+function open(string $uri, string $mode): ResourceHandle
```

Comment:
Since this one is internal, will we (besides stdin, stderr, stdout) also provide shortcut functions for any other wrapper out there and what with custom wrappers?

https://www.php.net/manual/en/wrappers.php

Reply:
honestly, i don't think we should support wrappers.

instead of creating wrappers, people should implement custom handles.